dataplanes/vpp/pkg/nsmutils/nsm-utils.go

```diff
@@ -91,3 +99,13 @@ func Ipv4prefixlength(value string) error {
 	}
 	return nil
 }
+
+func Empty(value string) error {
```

Comment:
I think there is `Any` validator function already, which allow any value to be specified as a parameter